x-pack/plugins/snapshot_restore/public/plugin.ts

```diff
@@ -27,61 +27,56 @@ export class Plugin {
         defaultMessage: 'Snapshot and Restore',
       }),
       order: 7,
-      url: `#${CLIENT_BASE_PATH}/repositories`,
+      url: `#${CLIENT_BASE_PATH}`,
     });
 
-    const unmountReactApp = (elem: Element | undefined | null): void => {
+    const unmountReactApp = (): void => {
+      const elem = document.getElementById(REACT_ROOT_ID);
```

Comment:
Are you sure about this refactor? We kept the reference to the `elem` outside so we could unmount the react app without losing the reference to the dom node (as Angular was creating a new `div` in the DOM)... 

Did you fully test this?

Reply:
yep. these changes were made due to the react app mounting twice when navigating away and coming back (React dev tools showed 2 apps)

Reply:
Ok.... then we might have the bug in our other apps as they are all using this pattern. I'll check.